Objects layers renamed to base file name instead of "topnode" & More then one file can be loaded in the Resource Window

herace opened the pull request:
- Objects layers renamed to base file name instead of "topnode"
- More then one file can be loaded in the Resource Window

A small update that i found useful when using AnimeEffects :) .
I think im close to figuring out how to add more then one object from the resource window but for now ill just give whatever i have since it speeds ups my workflow personally.

herace committed:
Resource Dialog Loads more then one file.

herace committed:
"create layer object" now titled file base name instead of "topnode"

Jose-Moreno commented:
@herace That's really nice! I think it was one of those things that had to be done :smile: 

FYI I suspect the project lead dev is either too busy with his life or has taken a "recess" for the time being from developing free stuff, if you know what I mean, as I stopped receiving updates for this app ever since December 2017.

If you're going to continue developing AE probably it is better to keep your fork up and if possible I'll gladly direct anyone that still uses the software and wants to develop it further to visit your repo for the time being :slightly_smiling_face:

herace commented:
@Jose-Moreno Dude thank you SO much for the feedback dude and also the current state of this repo. Ill try my very best to improve whatever i can :smile:

herace committed:
More then one layer can be created from resource list.

herace committed:
Merge pull request #1 from herace/dev

Create more then one new layers from the resource Dialog

larpon committed:
Fix a few compiler warnings

larpon committed:
* Add Qt::KeypadModifier to known modifier keys
* Add Rotate Canvas (Anti) Clockwise key bindings
* Start of WIP showing Key Bindings in tool tips

herace committed:
Merge pull request #2 from Larpon/fixes/fix_a_few_warnings

Fix a few compiler warnings

herace committed:
Merge pull request #3 from Larpon/feature/rotate-canvas-with-numpad

Add canvas rotation using NumPad + KeyPad numbers

larpon committed:
Fixes #4 and fixes saved tabs issue

larpon committed:
WIP: Fix old-style cast warnings

herace committed:
Merge pull request #6 from Larpon/fix/more-old-style-cast-warnings

WIP: Fix old-style cast warnings

herace committed:
Merge pull request #5 from Larpon/fix/saving-of-keypad-modifier-keys

Fixes saving of Keypad modifier keys in settings

larpon committed:
Add InfoLabelWidget

larpon committed:
Add and use priliminary InfoLabelWidget

larpon committed:
Add TimeFormat class

larpon committed:
Add General Setting for Time format display type. Use TimeFormat on Timeline header and In InfoLabelWidget

larpon committed:
Add conversion pseudo code

larpon committed:
Add HH:MM:SS:mmm time format option

larpon committed:
Add onApplicationSettingUpdated signal to MainMenuBar

larpon committed:
Add onApplicationSettingUpdated signal and signal trigger to TimeLineWidget

larpon committed:
Connect onApplicationSettingUpdated to InfoLabelWidget::onUpdate to get notified when some setting is updated

larpon committed:
Connect TimeLineWidget::triggerOnApplicationSettingUpdated to MainWindow::onApplicationSettingUpdated to be able to propergate setting changes to child widgets

larpon committed:
Add keymapping to tooltip for reseting the canvas and be a little more explicit in the tooltip variable names

larpon committed:
Add option to move canvas with middle-mouse button

larpon committed:
Fix a few warnings in core A to B* source files

herace committed:
Merge pull request #12 from Larpon/wip/fix-more-compiler-warnings

Fix a few warnings in core A to B* source files

herace committed:
Merge pull request #11 from Larpon/wip/add-more-keymappings-to-tooltips

Add one more keymapping to tooltip

larpon committed:
Merge branch 'master' into feature/timeline-information

larpon committed:
Rename InfoLabelWidget -> TimeLineInfoWidget

larpon committed:
Change "Time scale format" -> "Timeline format"

herace committed:
Merge pull request #10 from Larpon/feature/middle-mouse-canvas-movement

Add option to move canvas with middle-mouse button

larpon committed:
Merge branch 'master' into feature/timeline-information (canvas moving with middle mouse)

herace committed:
Merge pull request #8 from Larpon/feature/timeline-information

Add feature timeline information

larpon committed:
* Remove superfluous variables and static_casts
* Fix a few: "old-style cast..." and float vs. double conversions warning messages in touched files
* Move TimeFormat::remap to util::MathUtil::remap
* Add cycle function to util::MathUtil
* Expose private "maxFrame" member value via getter in time_Scaler
* Fix "SMPTE" and "Seconds : Frame" time formats to correctly start over when reaching end of scale (bug report via Discord by @JoseMoreno)

larpon committed:
Introduce themes directory

larpon committed:
Practice better code readability in classic theme

larpon committed:
Prepare theme addition to GUIResources class

larpon committed:
Fix GUIResources icon loading

larpon committed:
Introduce GUIResources::onThemeUpdated signal. Move widget stylesheet setting to onThemeUpdated slots

herace committed:
Merge pull request #13 from Larpon/hotfix/cycle-frames-time-codes

Fix `SMPTE` and `Seconds : Frame` time formats

herace committed:
Update README.md

Added chat board links on readme file.

herace committed:
Update README.md

herace committed:
Update README.md

larpon committed:
Merge branch 'master' into feature/theme-support

Fix conflict in src/ctrl/time/time_Renderer.cpp as the setting name has changed

larpon committed:
Add theme support for TimeLineEditorWidget

larpon committed:
Introduce theme::Theme class to get theme specific information from. Introduce theme::TimeLine proxy for sending Qt stylesheet changes to non-QWidget derived classes

azagaya committed:
Testing Travis CI

fixed root directory in travis.yml

Added target

Updated travis

Updated location of pro

Trying to cp binary, as i cannot manage to install it

Fixed app name in travis.yml

Changed app name

Added data to appdir

Added -R to cp

herace committed:
Merge pull request #15 from azagaya/master

Adding Travis CI with appimage deployment

larpon committed:
Add preliminary light icons to default theme

larpon committed:
Remove stray .directory file

larpon committed:
Let ObjectTreeWidget use system palette defined color for background

larpon committed:
Add auto detection of icon color. Make icons change on theme change at runtime

herace committed:
Merge pull request #16 from Larpon/feature/theme-support

Add theme support

FoxyHawk committed:
Fix missing dependancy for Qt5.15

FoxyHawk committed:
Fix deprecated functions

herace committed:
Merge pull request #18 from FoxyHawk/qt515_support

Qt515 support

freddii committed:
added missing build instruction for linux

herace committed:
Merge pull request #20 from freddii/master

added missing build instruction for linux

herace committed:
Update README.md

herace committed:
Update README.md

Updated Debian based linux distros installation instructions.
Added Arch based linux distros installation instructions.

herace committed:
Update README.md

removed "sudo" on qmake and make steps as WolfyHawk suggested

herace committed:
Update README.md
